cfpb/generator-cf
# Write deps to bower.json

Writing CF component dependencies to the bower file

## Additions

None

## Removals

None

## Changes

- updated index.js to get latest release from GitHub api
- updated `_bower.json` template to write user chosen deps

## Testing

- fetch and run `npm link`
- run `mocha`
- check /test/temp/bower.json for list of deps with version numbers

## Review

- @contolini 
- @ascott1 
- @Scotchester 
- @KimberlyMunoz
- @himedlooff 

## Preview

![screen shot 2015-03-25 at 2 46 59 pm](https://cloud.githubusercontent.com/assets/1280430/6832567/d07e4712-d2fd-11e4-9a65-cb792593cea4.png)


## Notes

- pings GitHub tags endpoint and grabs most recent release tag
- sets the repo name and version in bower template
- waits to call tag endpoint until user has set preferred components to avoid erroneous API calls
- Related to #30
